anta/cli/get/commands.py

```diff
@@ -61,7 +61,30 @@ def from_cvp(inventory_directory: str, cvp_ip: str, cvp_username: str, cvp_passw
         # Get devices under a container
         logger.info(f"Getting inventory for container {cvp_container} from {cvp_ip}")
         cvp_inventory = clnt.api.get_devices_in_container(cvp_container)
-    create_inventory(cvp_inventory, out_dir, cvp_container)
+    create_inventory_from_cvp(cvp_inventory, out_dir, cvp_container)
+
+
+@click.command(no_args_is_help=True)
+@click.option("--ansible-group", "-g", default=None, help="Ansible group to filter", type=str, required=False)
+@click.option("--ansible-inventory", "-i", default=None, help="Path to your ansible inventory file to read", type=click.File())
+@click.option("--output", "-o", default='inventory-ansible.yml', help="Path to save inventory file", type=click.File(mode='w', lazy=True))
+@click.option("--inventory-directory", "-d", default='.', help="Directory to save inventory file", type=click.Path())
+def from_ansible(inventory_directory: str, output: str, ansible_inventory: str, ansible_group: str) -> None:
+    """Build ANTA inventory from an ansible inventory YAML file"""
+    logger.info(f"Building inventory from ansible file {ansible_inventory}")
+
+    # Create output directory
+    cwd = os.getcwd()
```

Comment:
Do not assume the user wants the output in the current directory. Ask the user to provide a path and create the missing folder.